plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
```

Comment:
missing copyrights (maybe that's why PR fails CI)

Reply:
no no it something related to the connection from the runner to the machine, you can see the git action works.